source/MaterialXGenShader/Util.cpp

```diff
@@ -520,7 +520,7 @@ bool elementRequiresShading(const TypedElementPtr element)
             colorClosures.count(elementType) > 0);
 }
 
-void findRenderableElements(const DocumentPtr& doc, std::vector<TypedElementPtr>& elements)
+void findRenderableElements(const DocumentPtr& doc, std::vector<TypedElementPtr>& elements, bool includeReferencedGraphs)
```

Comment:
Issue was found in connected graphs to a shader. Adding this option so it's possible to consider these as renderable as it's usefull to examine connected upstream graphs.